_emails/newsletter-2019-03.html

```diff
@@ -0,0 +1,72 @@
+---
+title: Origami Newsletter, March 2019
+companion_post_url: https://origami.ft.com/blog/2019/03/29/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+
+    This issue features the new Origami website with a new design and new documentation; a highlight of the improved service that the Polyfill Service provides, and an explanation of how ECMAScript Modules broke our components.
+
+    {% endcapture %}
+{% include email/tldr.html content=text %}
+
+
+<!-- Top 3 things -->
+{% include email/h2.html content="Top three things" %}
+
+{% capture text %}These are some of the bigger things we've worked on or released over the last month.{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Origami Website Launch" %}
+{% capture text %}
+
+    This month we've successfully launched the <a href="https://origami.ft.com/" target="_blank">new Origami website</a>. It is officially out of beta, and implements all of the new features of o-layout, which has unified the design across our documentation and our registry. In addition to the visual changes, this launch is packed full of new documentation and a more digestible spec. You can find more detailed information about the launch in <a href="https://origami.ft.com/blog/2019/03/11/site-update">this blog post</a>. As always, we welcome any feedback or questions you might have about the new website's content and UI.
+
+{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+<!-- Thing 2 -->
+{% include email/h3.html content="Polyfills served" %}
+{% capture text %}
+
+    Three years ago, the <a href="https://polyfill.io" target="_blank">Polyfill Service</a> served 400 million polyfills per month. Today, it serves <strong>5.6 billion</strong> every week, which is about <strong>800 million</strong> <em>every day</em> 💪.
```

Comment:
```suggestion
    Three years ago, the <a href="https://polyfill.io" target="_blank">Polyfill Service</a> served 400 million polyfills per month. Today, it serves <strong>800 million</strong> <em>every day</em> 💪.
```
I don't think we need to mention the week total.